### PR TITLE
Adding Contact us page to Edge extensions documentation

### PR DESCRIPTION
**Pull request by gebansal**

### CODE REVIEW

**Comment by gebansal** [comment]
Acrolinx score - 92

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ms.topic: support
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, support
+---
+
+# Contact Edge extension support
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification write to us at support [here][ExtensionsSupportForm].
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:16]:
```suggestion
1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
```

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+1. If you have any issues or feedback regarding your publishing experience via Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:17]:
```suggestion
1. If you have any issues or feedback regarding your publishing experience using Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
```

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+1. To feature your extension in a collection on the Edge Add-ons home page - submit a request using [this][CollectionsForm] form.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:18]:
```suggestion
1. To feature your extension in a collection on the Edge Add-ons home page, submit your request using the [Submit a request to add an extension to the collections on the MS Edge Add-ons home page][CollectionsForm] form. 
```

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification write to us at support [here][ExtensionsSupportForm].
+1. If you have any issues or feedback regarding your publishing experience via Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
+1. To feature your extension in a collection on the Edge Add-ons home page - submit a request using [this][CollectionsForm] form. 
+
+
+## Report copyright or trademark infringement
+If you think an item in the Edge add-ons store violates a copyright or trademark, fill out [this][EdgeInfringmentReport] form. Our team will review your report and take any necessary action.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:22]:
```suggestion
If you think an item in the Edge add-ons store violates a copyright or trademark, fill out the [Notices of Infringement][EdgeInfringmentReport] form. Our team will review your report and take any necessary action.
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+title: Contact Edge extensions support
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/19/2021
+ms.topic: support
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, support
+---
+
+# Contact Edge extension support
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:12]: It should be "Microsoft Edge"

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:16]: Suggest we make this a little more clear. 

If you have submitted an extension via the Partner Center and have a question related to its review/certification status, please submit a New Support Request.

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+1. If you have any issues or feedback regarding your publishing experience using Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:17]: Suggest we make a small change here and first redirect to our publishing guide. 

If you are facing any issues while submitting your extension to the Partner Center review the tutorial to Publish your extensions. If that does not answer your question or you have a feedback regarding your publishing experience write to us at...

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+1. To feature your extension in a collection on the Edge Add-ons home page, submit your request using the [Submit a request to add an extension to the collections on the MS Edge Add-ons home page][CollectionsForm] form.
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:18]: Add another section here:

If your extension is not working as expected with Microsoft Edge or you have a question regarding how to develop an extension, check out these links: Supported APIs for Microsoft Edge extensions and Port Chromium extensions to Microsoft Edge. For any other questions, write to us at xxxx.

File: microsoft-edge/extensions-chromium/publish/contact-us.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+## Contact us for any issues or feedback
+
+1. For any issues related to certification, please submit a [New Support Request][ExtensionsSupportForm].
+1. If you have any issues or feedback regarding your publishing experience using Partner Center, write to us at [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
+1. To feature your extension in a collection on the Edge Add-ons home page, submit your request using the [Submit a request to add an extension to the collections on the MS Edge Add-ons home page][CollectionsForm] form. 
+
+
+## Report copyright or trademark infringement
+If you think an item in the Edge add-ons store violates a copyright or trademark, fill out the [Notices of Infringement][EdgeInfringmentReport] form. Our team will review your report and take any necessary action.
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/contact-us.md:22]: It should be "Microsoft Edge add-ons"

**Review by jm-trd-ms — CHANGES_REQUESTED** [review]
Please change the file name and the tile in the TOC.

**Review by Reezaali — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**